src/App/Game/Game.tsx

```diff
@@ -123,7 +123,9 @@ export default class Game extends React.Component<IGameProps, {}> {
         clientConfig.ai = state.ai.bgioAI(aiLevel);
       }
       if (this.mode === GameMode.OnlineFriend) {
-        clientConfig.multiplayer = true;
+        const server = typeof window !== 'undefined' ?
+          process.env.BGIO_SERVER_URL || `${window.location.hostname}:8001` : 'localhost:8001';
```

Comment:
When would the window be undefined?

Reply:
Hrmph.  I think I ran into an issue testing with Jest when I added this, but I don't remember/can't reproduce now, so removing this.

Reply:
Aha.  It was for SSR.

```
ReferenceError: window is not defined
      at Game.render (webpack:///./src/App/Game/Game.tsx?:132:58)
      at /appdata/node_modules/react-tree-walker/dist/react-tree-walker.js:211:33
      at /appdata/node_modules/react-tree-walker/dist/react-tree-walker.js:141:36
```